examples/self_supervised/common.py

```diff
@@ -45,3 +47,30 @@ def add_arguments(parser) -> None:
     parser.add_argument(
         "--learning-rate", default=0.001, type=float, help="Learning rate for optimizer"
     )
+
+
+class ContrastiveModel(torch.nn.Module):
```

Comment:
could you please update the code for other `self_supervised` examples with such a model?

Reply:
[Done](https://github.com/catalyst-team/catalyst/pull/1305/commits/95f2b47750afe14fe669c3a96f95c33ebf09c35a)